analysis/generate_report.py

```diff
@@ -155,10 +155,10 @@ def generate_report(experiment_names,
     data_path = os.path.join(report_directory, 'data.csv.gz')
     if from_cached_data and os.path.exists(data_path):
         experiment_df = pd.read_csv(data_path)
+        description = "from cached data"
```

Comment:
at some point, i will fix this, not needed for this cl. i think description is there in experiment_df.